### PR TITLE
claim bribes from voter script

### PR DESCRIPTION
**Pull request by jalbrekt85**
handles #709

```coffeescript
brownie run badger/claim_hh
```

### CODE REVIEW

File: scripts/badger/claim_hh.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    aggregate = {"tokens": [], "amounts": []}
+    for item in data:
+        aggregate["tokens"].append(item["token"])
+        aggregate["amounts"].append(item["claimMetadata"]["amount"])
+
+    voter.take_snapshot(tokens=aggregate["tokens"])
+    destination.take_snapshot(tokens=aggregate["tokens"])
+
+    rewards.claim(
```
Comment (petrovska-petro) [inline scripts/badger/claim_hh.py:25]: i will say given that we have this [method](https://github.com/Badger-Finance/badger-multisig/blob/main/great_ape_safe/ape_api/badger.py#L154) in the badger class. will advise to leverage and allow passing an address for where bribes will be claimed

Comment (jalbrekt85) [reply scripts/badger/claim_hh.py:25]: good idea

Comment (jalbrekt85) [reply scripts/badger/claim_hh.py]: updated in [7d83c21](https://github.com/Badger-Finance/badger-multisig/pull/726/commits/7d83c21c22e4cc69971be521696cc371a355d3d1), allows address to be passed into `claim_bribes_hidden_hands` but claims from gravi aura strat by default as to not break other scripts

**Review by petrovska-petro — APPROVED** [review]
nice ! 

ready to recycle those badger bribes 🔥

**Review by gosuto-inzasheru — APPROVED** [review]
rock n roll

**Pull request merged**